client/flutter/lib/pages/main_pages/home_page_router.dart

```diff
@@ -0,0 +1,59 @@
+import 'package:firebase_analytics/firebase_analytics.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:who_app/constants.dart';
+import 'package:who_app/pages/main_pages/check_up_page.dart';
+import 'package:who_app/pages/main_pages/home_page.dart';
+import 'package:who_app/pages/main_pages/learn_page.dart';
+import 'package:who_app/pages/main_pages/settings_page.dart';
+import 'package:who_app/pages/main_pages/stats_page.dart';
+
+class HomePageRouter extends StatelessWidget {
+  final FirebaseAnalytics analytics;
+
+  HomePageRouter(this.analytics);
+
+  _logAnalyticsEvent(String name) async {
+    await analytics.logEvent(name: name);
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return CupertinoTabScaffold(
+      tabBuilder: (BuildContext context, int index) {
+        switch (index) {
+          case 0:
+            return HomePage();
+          case 1:
+            return StatsPage();
+          case 2:
+            return LearnPage();
+          case 3:
+            return CheckUpPage();
+          case 4:
+            return SettingsPage();
+
+          default:
+            return null;
+        }
+      },
+      tabBar: CupertinoTabBar(
+        inactiveColor: Colors.black,
+        activeColor: Constants.accent,
+
+        items: [
+          BottomNavigationBarItem(
+              icon: Icon(CupertinoIcons.home), title: Text("Home")),
```

Comment:
please mark a TODO & open an issue for localizing the text here

```suggestion
              // TODO: localize title strings
              icon: Icon(CupertinoIcons.home), title: Text("Home")),
```